### DISCUSSION TITLE
download is being cancelled and cannot be resumed when an update is coming up

### DISCUSSION THREAD

**Discussion by pa12093** [aclap-dev/video-downloadhelper#170] [Help and Q&A]
### Your question

download is being force closed and cannot be resumed when an update is coming, which a user needs to restart the entire download process, which is not smart at all

just like a phone when you are playing a game then an update is coming and the phone is self rebooting to carry out updating process. the user has to restart the game at the beginning after the phone was updated

any developers can put yourselves in my shoes? or you think this is a clever design?

### Addon & CoApp version and details

firefox, 8.2.0.24

**Comment by paulrouget** [comment]
I agree with you. Lot of people asked for it. That's something we are working on for the version 8.3.
